nixpkgs_review/review.py

```diff
@@ -312,12 +459,51 @@ def list_packages(path: str, system: str, check_meta: bool = False) -> List[Pack
     ]
     if check_meta:
         cmd.append("--meta")
+
+    use_cache = False
```

Comment:
Can you leave this disk cache out for now of this PR and make a new one for it?

Reply:
yes

i used diskcache to make development faster
on my machine, one `nix-env -qaP` takes around 400 seconds
which is a killjoy for development

diskcache could be hidden behind a

```py
if os.environ.get("NIXPKGS_REVIEW_DISKCACHE"):
    # ...
```

problem with caching: filepaths
nixpkgs-review uses different temporary file paths on every run
so the cache entries are always slightly invalid
i have not-yet checked if that has a negative impact on package diffing
normalizing filepaths should be easy